src/general/NewGameSettings.cs

```diff
@@ -144,6 +144,9 @@ public partial class NewGameSettings : ControlWithInput
     [Export]
     public NodePath GameSeedAdvancedPath = null!;
 
+    [Export]
+    public NodePath WorldSizeButtonPath = null!;
```

Comment:
Could you use the new Godot 4 approach here where the `worldSizeButton` is directly given the `Export` attribute? As we are slowly trying to move away from this old NodePath approach.